tests/test_ingest.py

```diff
@@ -40,7 +40,7 @@
 from mock_data.matrix_mtx.gene_model_0 import expected_model
 from gcp_mocks import mock_storage_client, mock_storage_blob
 
-sys.path.append('../ingest')
+sys.path.append("../ingest")
```

Comment:
Quote noise like this makes it hard to see relevant differences.  

It could be avoided by using `--skip-string-normalization` in one's IDE, as in [our Git hook](https://github.com/broadinstitute/scp-ingest-pipeline/blob/master/.pre-commit-config.yaml#L12).

Or we could remove that Black argument from our hook and our IDE configurations, and double-quote all strings in Python.  This would increase our Python's difference from our JavaScript, but ensuring readable diffs with minimal IDE configuration seems more important.

Eno, Jean, and I chatted and we lean toward removing `--skip-string-normalization`.